### ISSUE TITLE
Add directories doctrees and html to gitignore 

### ISSUE THREAD

**Issue by vipulgupta2048** [godiard/help-activity#45]
While executing the command `make html` a lot of static HTML files are being generated which should be ignored by `.gitignore` but are not. Same goes the doctrees that Sphinx builds, itself. 

I think we should add them to the .gitignore to save future push blunders. Suggestions appreciated on the same are appreciated.  
(Personally, I was just about to push 532 changed files :man_facepalming: because ./doctrees and html/* were not mentioned in .gitignore. Had to double check the code in the repo to see, if they were even present or not. Phew  )

**Comment by quozl** [comment]
That's odd, `html/*.html` is already in `.gitignore`.  Please check?

**Comment by vipulgupta2048** [comment]
@quozl  Yes it is, what I am referring to is the static files that being built. Refer to the screenshot. 
It should be `html/*` rather than `html/*.html`

![screenshot_2018-04-09_11-42-47](https://user-images.githubusercontent.com/22801822/38482012-2c256baa-3beb-11e8-9be3-ec5798c019e1.png)

Namely /es , /_static and /_images

**Comment by quozl** [comment]
Thanks.  Presumably the new version of Sphinx is root cause.

Meanwhile, please consider more careful methods for adding files to git index.  My preference is magit in emacs.

**Comment by vipulgupta2048** [comment]
Making a PR. How about doctrees, add them too?
They just take way too much space.

> Meanwhile, please consider more careful methods for adding files to git index. My preference is magit in emacs.

Googled it, I like it. Gotta look for something that would help me. I use Fish and Geany btw.

**Comment by quozl** [comment]
In `.gitignore` should be patterns to match the files made by build that do not need to be added to git.

**Comment by vipulgupta2048** [comment]
@quozl That's right, but what I actually meant was. doctrees/ directory is created every time by Sphinx upon generating HTML documents is not even present in the [code repository](https://github.com/godiard/help-activity) in the first place. My query is why it is not added to the `.gitignore` since we won't be needing that and they are quite heavy in size (2.3 MB) 
I hope I have made myself clear. Please advise on the same. Here's a screenshot for further reference.

![screenshot_2018-04-10_05-36-26](https://user-images.githubusercontent.com/22801822/38529214-278076e4-3c81-11e8-9e8c-e60df0b40d64.png)

**Comment by quozl** [comment]
You ask why is `doctrees` not in `.gitignore`?  Because 9e498fe4ca4728818f886993ec8ee0c773f59451 and 2a76eb4306bcbcf96949ab44e8fc49cc872407ee did not add it.  `doctrees` exists because of 7216fca2524aad4838b72efb55de25292877d942.  Not caused by Sphinx; is a parameter for Sphing instead.

I use a text only mail client.  My filesystem UIs are bash and emacs, so screenshots of graphical filesystem managers are mostly meaningless to me.  Better would be backtick quoted output from `git status`.  Like this;

```
host:~/Help$ git clean -dfx
host:~/Help$ make html
...
host:~/Help$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Untracked files:
  (use "git add <file>..." to include in what will be committed)

        doctrees/
        html/.buildinfo
        html/_images/
        html/_static/
        html/objects.inv
        html/searchindex.js

nothing added to commit but untracked files present (use "git add" to track)
host:~/Help$ █
```

Above shows the files created by the `html` target that are not included in `.gitignore`.  This confirms the issue you have reported.

**Comment by vipulgupta2048** [comment]
Closing the issue, as #48 fixes it.
